src/workflow_components/composite_action.py

```diff
@@ -27,6 +28,42 @@ def get_or_create_composite_action(path: str) -> "CompositeAction":
     return obj
 
 
+class CompositeActionInput(GraphObject):
+    __primarykey__ = "_id"
+
+    _id = Property()
+    name = Property()
```

Comment:
I suggest to change it to param

Reply:
I don't think so. If it was a property in composite action, I would agree, but the `name` refers to the input. So it is the name of the input that makes sense. The Input is a "param" by itself.

Reply:
It also makes more sense when filtering: `cai: CompositeActionStep` -> `cai.name` against `cai.param` which could be confusing.